data/35e/wizards_of_the_coast/supplement/weapons_of_legacy/wl_legacyitem_template.lst

```diff
@@ -0,0 +1,834 @@
+CATEGORY=Internal|LegacyItemName ~ Tracker.MOD		BONUS:ABILITYPOOL|LegacyItemName Personal Cost|1|PREMULT:2,[PREABILITY:1,CATEGORY=Feat,Least Legacy ~ LegacyItemName],[PREPCLEVEL:MIN=5]		BONUS:ABILITYPOOL|LegacyItemName Personal Cost|1|PREMULT:2,[PREABILITY:1,CATEGORY=Feat,Least Legacy ~ LegacyItemName],[PREPCLEVEL:MIN=6]		BONUS:ABILITYPOOL|LegacyItemName Personal Cost|1|PREMULT:2,[PREABILITY:1,CATEGORY=Feat,Least Legacy ~ LegacyItemName],[PREPCLEVEL:MIN=7]		BONUS:ABILITYPOOL|LegacyItemName Personal Cost|1|PREMULT:2,[PREABILITY:1,CATEGORY=Feat,Least Legacy ~ LegacyItemName],[PREPCLEVEL:MIN=8]		BONUS:ABILITYPOOL|LegacyItemName Personal Cost|1|PREMULT:2,[PREABILITY:1,CATEGORY=Feat,Least Legacy ~ LegacyItemName],[PREPCLEVEL:MIN=9]		BONUS:ABILITYPOOL|LegacyItemName Personal Cost|1|PREMULT:2,[PREABILITY:1,CATEGORY=Feat,Least Legacy ~ LegacyItemName],[PREPCLEVEL:MIN=10]
+CATEGORY=Internal|LegacyItemName ~ Tracker.MOD		BONUS:ABILITYPOOL|LegacyItemName Personal Cost|1|PREMULT:2,[PREABILITY:1,CATEGORY=Feat,Lesser Legacy ~ LegacyItemName],[PREPCLEVEL:MIN=11]		BONUS:ABILITYPOOL|LegacyItemName Personal Cost|1|PREMULT:2,[PREABILITY:1,CATEGORY=Feat,Lesser Legacy ~ LegacyItemName],[PREPCLEVEL:MIN=12]		BONUS:ABILITYPOOL|LegacyItemName Personal Cost|1|PREMULT:2,[PREABILITY:1,CATEGORY=Feat,Lesser Legacy ~ LegacyItemName],[PREPCLEVEL:MIN=13]		BONUS:ABILITYPOOL|LegacyItemName Personal Cost|1|PREMULT:2,[PREABILITY:1,CATEGORY=Feat,Lesser Legacy ~ LegacyItemName],[PREPCLEVEL:MIN=14]		BONUS:ABILITYPOOL|LegacyItemName Personal Cost|1|PREMULT:2,[PREABILITY:1,CATEGORY=Feat,Lesser Legacy ~ LegacyItemName],[PREPCLEVEL:MIN=15]		BONUS:ABILITYPOOL|LegacyItemName Personal Cost|1|PREMULT:2,[PREABILITY:1,CATEGORY=Feat,Lesser Legacy ~ LegacyItemName],[PREPCLEVEL:MIN=16]
+CATEGORY=Internal|LegacyItemName ~ Tracker.MOD		BONUS:ABILITYPOOL|LegacyItemName Personal Cost|1|PREMULT:2,[PREABILITY:1,CATEGORY=Feat,Greater Legacy ~ LegacyItemName],[PREPCLEVEL:MIN=17]		BONUS:ABILITYPOOL|LegacyItemName Personal Cost|1|PREMULT:2,[PREABILITY:1,CATEGORY=Feat,Greater Legacy ~ LegacyItemName],[PREPCLEVEL:MIN=18]		BONUS:ABILITYPOOL|LegacyItemName Personal Cost|1|PREMULT:2,[PREABILITY:1,CATEGORY=Feat,Greater Legacy ~ LegacyItemName],[PREPCLEVEL:MIN=19]		BONUS:ABILITYPOOL|LegacyItemName Personal Cost|1|PREMULT:2,[PREABILITY:1,CATEGORY=Feat,Greater Legacy ~ LegacyItemName],[PREPCLEVEL:MIN=20]
+CATEGORY=Internal|LegacyItemName ~ Tracker.MOD		DEFINE:LegacyItemShortNameTotalEnhancement|0			BONUS:VAR|LegacyItemShortNameTotalEnhancement|min(5,(LegacyItemShortNameEnhancementBonus+(count("ABILITIES","CATEGORY=Internal","NAME=LegRepEnh1 (LegacyItemName)")*1)+(count("ABILITIES","CATEGORY=Internal","NAME=LegRepEnh2 (LegacyItemName)")*2)+(count("ABILITIES","CATEGORY=Internal","NAME=LegRepEnh3 (LegacyItemName)")*3)+(count("ABILITIES","CATEGORY=Internal","NAME=LegRepEnh4 (LegacyItemName)")*4)+(count("ABILITIES","CATEGORY=Internal","NAME=LegRepEnh5 (LegacyItemName)")*5)))
```

Comment:
Has this been tested and determined to work?

Reply:
I have built various characters with each item to test that cost pools get points at the correct levels. 
However, if it is better to use a single bonus variable rather that multiple bonuses with PREMULT, i could rework that. Each legacy feat could set a "hasLeast/Lesser/Greater" variable to 1 and the grant pool would have to look like ...+((CL>=7)*hasLeastLegacy*PoolBonusAt7)+...

Reply:
About the "...LegacyItemShortNameTotalEnhancement|min(5,(LegacyItemShortNameEnhancementBonus+(count(..." part, this is for the very niche case of replacing an item ability in order to increase the enhancement bonus of the item. Doing this reverse lookup was the only way I could think off to include the bonus automatically, but even that has problems, since the user can chose to add an armor/weapon special ability instead of the enhancement bonus.

The sensible way to add the special ability is by making a new custom item that includes it. The same could be done with the enhancement bonus. By adding a short note to the replacement ability that players need to fix their item manually, we could get rid of this calculation.